paket.lock

```diff
@@ -65,12 +65,11 @@ NUGET
       Nancy (>= 1.2.0)
     Newtonsoft.Json (6.0.8)
     NuGet.CommandLine (2.8.5)
-    Nuget.Core (2.8.5)
+    NuGet.Core (2.8.5)
```

Comment:
It this really causing a Travis build failure. Yikes. That is the real name of the package and is what Paket-Install 1.18.0 set it to.